core/src/main/scala/janstenpickle/scala/syntax/syntax.scala

```diff
@@ -24,95 +25,122 @@ object catsconversion {
+  val shit = (
+    Result.fail[List[String], Int](List("sdfsf")
+  )
+  |@| Result.fail[List[String], Int](List("3423rwfef"))
+  |@| Result.ok[List[String], String]("sdfsf")).map((x, y, z) ⇒ (x, y, z))
 }
 
-object option {
+object CatsOption {
   implicit class ToTuple[T](opt: Option[T]) {
     def toMap(key: String): Map[String, T] =
-      opt.fold[Map[String, T]](Map.empty)(v => Map(key -> v))
+      opt.fold[Map[String, T]](Map.empty)(v ⇒ Map(key -> v))
   }
 }
 
-object asyncresult {
-  implicit class FutureToAsyncResult[T](future: Future[T])(implicit ec: ExecutionContext) {
-    def toAsyncResult: AsyncResult[String, T] = AsyncResult(future.map(Result.ok))
+object CatsAsyncResult {
+  implicit class FutureToAsyncResult[T](future: Future[T])
+  (implicit ec: ExecutionContext) {
+    def toAsyncResult: AsyncResult[String, T] = AsyncResult(
+      future.map(Result.ok)
+    )
   }
 
-  implicit class ReqToAsyncResult(req: Req)(implicit ec: ExecutionContext) {
+  implicit class ReqToAsyncResult(req: Req)
+  (implicit ec: ExecutionContext) {
     def toAsyncResult: AsyncResult[String, Response] = Http(req).toAsyncResult
   }
 
-  implicit def toAsyncResult[T](future: scala.concurrent.Future[T])(implicit ec: ExecutionContext): AsyncResult[String, T] =
+  implicit def toAsyncResult[T](future: scala.concurrent.Future[T])
+  (implicit ec: ExecutionContext): AsyncResult[String, T] =
     future.toAsyncResult
 }
 
-object vaultconfig {
+object CatsVaultConfig {
 
   final val VaultTokenHeader = "X-Vault-Token"
 
   implicit class RequestHelper(config: VaultConfig) {
-    def authenticatedRequest(path: String)(req: Req => Req)
-                            (implicit ec: ExecutionContext): AsyncResult[String, Req] =
-      config.token.map[Req](token =>
+    def authenticatedRequest(path: String)(req: Req ⇒ Req)
+    (implicit ec: ExecutionContext): AsyncResult[String, Req] =
+      config.token.map[Req](token ⇒
         req(config.wsClient.path(path).setHeader(VaultTokenHeader, token))
       )
   }
 }
 
-object json {
-  import catsconversion._
+object CatsJson {
+  import CatsConversion._
 
   implicit class JsonHandler(json: AsyncResult[String, Json]) {
-    def extractFromJson[T](jsonPath: HCursor => ACursor = _.acursor)
-                          (implicit decode: Decoder[T], ec: ExecutionContext): AsyncResult[String, T] =
-      json.flatMapR(j => decode.tryDecode(jsonPath(j.hcursor)).leftMap(_.message))
+    def extractFromJson[T](jsonPath: HCursor ⇒ ACursor = _.downArray)
+    (
+      implicit decode: Decoder[T],
+      ec: ExecutionContext
+    ): AsyncResult[String, T] =
+      json.flatMapR(j ⇒ decode.tryDecode(
+        jsonPath(j.hcursor)
+      ).leftMap(_.message))
   }
 }
 
-object response {
-  import catsconversion._
-  import json._
+object CatsResponse {
```

Comment:
@janstenpickle, the reason I have add the `Cats` prefix is because compile error like:
```
[error] /home/eric/Work/try-out/scala/scala-vault/core/src/main/scala/janstenpickle/scala/syntax/syntax.scala:3: imported `Response' is permanently hidden by definition of object Response in package syntax
[error] import com.ning.http.client.Response 
```
And I think it's more about style of naming, we can always change scalastyle config to allow lowercase class name. 

Thanks